lib/config/presets/internal/default.ts

```diff
@@ -586,6 +586,12 @@ export const presets: Record<string, Preset> = {
     separateMajorMinor: true,
     separateMultipleMajor: true,
   },
+  separateMultipleMinorReleases: {
+    description:
+      'Separate each `minor` version of dependencies into individual branches/PRs.',
+    separateMinorPatch: true,
```

Comment:
I think it still works without this, but just has a different branch name?

Reply:
ah yeah, just tested and does seem to work without it.
```suggestion
```